library/core/src/str/validations.rs

```diff
@@ -271,3 +277,28 @@ pub const fn utf8_char_width(b: u8) -> usize {
 
 /// Mask of the value bits of a continuation byte.
 const CONT_MASK: u8 = 0b0011_1111;
+
+#[cfg(kani)]
+#[unstable(feature = "kani", issue = "none")]
+pub mod verify {
+    use super::*;
+
+    #[kani::proof]
+    #[kani::unwind(8)]
```

Comment:
Is the unwind necessary?

Reply:
No, I think it is accidentally copied from another harness.